ci/install_deps.sh

```diff
@@ -4,17 +4,23 @@ set -o nounset
 set -o xtrace
 
 DEPS=(
-  build-essential git gdb valgrind cmake rpm \
-  python-dev libcap-dev python-pip python-virtualenv \
+  build-essential git gdb valgrind cmake rpm file
+  libcap-dev python-dev python-pip python-setuptools
   hardening-includes gnupg
 )
 
 if [[ -z "${ARCH_SUFFIX-}" ]] || [[ "$ARCH_SUFFIX" = "amd64" ]]; then
```

Comment:
I thought about making this whole section into a `case` statement instead, but didn't want this PR to change _too_ many lines. :innocent:

(Happy to update if that's something that'd be helpful/desirable!)